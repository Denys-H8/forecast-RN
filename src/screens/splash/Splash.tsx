import React from 'react';
import { View } from 'react-native';

import { styles } from './splash.styles';
import { CustomText } from 'src/components';

const Splash: React.FC = () => (
  <View style={styles.container}>
    <CustomText style={styles.text}>WEATHER APP</CustomText>
  </View>
);

export default Splash;
