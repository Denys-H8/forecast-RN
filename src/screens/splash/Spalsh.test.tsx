import React from 'react';
import { render } from '@testing-library/react-native';
import Splash from './Splash';

describe('Splash ', () => {
  test('should render Splash', () => {
    const { toJSON } = render(<Splash />);

    expect(toJSON()).toMatchSnapshot();
  });
});
