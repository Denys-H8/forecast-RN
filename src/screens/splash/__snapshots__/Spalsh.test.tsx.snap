// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash  should render Splash 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#191C39",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "WorkSans-Regular",
        },
        Object {
          "color": "#fff",
          "fontFamily": "WorkSans-SemiBold",
          "fontSize": 64,
          "lineHeight": 75,
          "textAlign": "center",
        },
      ]
    }
  >
    WEATHER APP
  </Text>
</View>
`;
